install

```diff
@@ -34,9 +28,10 @@ verbose=0
 #   print usage with examples.
 show_help() {
 	cat <<EOF
-install -p /usr/local/bin
+install -P /usr/local
```

Comment:
let's add also a `--prefix` example to be more clear

Reply:
Changed the example to only `--prefix`.